raymon/tests/conftest.py

```diff
@@ -44,3 +49,21 @@ def envsecretfile(tmp_path):
     with open(tmp_file, "w") as f:
         f.write("client_secret")
     return tmp_file
+
+
+@pytest.fixture
+def images(dpath="raymon/tests/sample_data", lim=10):
+    files = glob.glob(dpath + "/*.jpeg")
+    images = []
+    for n, fpath in enumerate(files):
+        if n == lim:
+            break
+        img = Image.open(fpath)
+        img.thumbnail(size=(500, 500))
+        images.append(img)
+    return images
+
+
+@pytest.fixture
+def save_profile_model_path(tmp_path):
+    return tmp_path
```

Comment:
What is this supposed to do? Why can you not just use the fixture that is already there (tmp_path) wherever you would use save_profile_model_path?

Reply:
I supposed "tmp_path" can be generated in only pytest.fixture
But I noticed that I can use directly it in a test_function. 
`def test_profile_inputComponent(images, tmp_path):`